ush/prdgen/rrfs_prdgen_faa_subpiece.sh

```diff
@@ -12,6 +12,8 @@ ififip=$5
 aviati=$6
 COMOUT=$7
 
+USHrrfs=$USHdir/prdgen
```

Comment:
I think USHrrfs should be defined somewhere else before this script, possibly in the J-job.  @MatthewPyle-NOAA where do you think makes the most sense?

Reply:
@BenjaminBlake-NOAA I agree it should be defined at the J-job level

Reply:
This is a local variable. I am seeing other ush subdirectories are defined in the same way. For example:
USHradmon in both ush/rrfs_radmon/exrrfs_verfrad.sh and ush/rrfs_radmon/exnam_verfrad.sh
and utildir in ush/Init.sh.

So I am not sure whether it should be defined at J-job level.

Reply:
@YaliMao-NOAA - defining it in the J-job is the EE2 standard.  Top of page 5 of https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf.